### PR TITLE
UI: Correctly set 'shell/open/command; registry key for file associations

### LINKED ISSUE #5243: [Bug] Windows - "Install file types" puts the wrong data in the "Type" field
### Description of the issue

Installed file types have the wrong data in the "Type" field.  Instead of showing "Ryujinx file" or "Nintendo Switch Data File" or anything of the sort, it instead displays the command that gets executed: 
`"<path to ryujinx.exe" "%1"` 

### Reproduction steps

In version 1.1.869 (it could exist in previous): 
From the menu bar, select Tools
-> Manage File Types
--> Install file types

### Log file

Oddly this generation produces ZERO log entries

### OS

Windows 10

### Ryujinx version

1.1.869

### Game version

_No response_

### CPU

_No response_

### GPU

_No response_

### RAM

_No response_

### List of applied mods

_No response_

### Additional context?

I haven't tested whether file associations work on other OSes, only Windows.

### PR DESCRIPTION
**Pull request by mmdurrant**
Fixes #5243 by correctly setting the registry key for the open command.  

Also wraps IDisposable resources in `using` blocks.

### CODE REVIEW

**Review by riperiperi — APPROVED** [review]
Looks fine here, nice one. There might be another issue with file type associations, as when I try to uninstall the file types on Windows 11 I get an error message saying that it failed.

**Review by AcK77 — APPROVED** [review]

**Comment by mmdurrant** [comment]
> Looks fine here, nice one. There might be another issue with file type associations, as when I try to uninstall the file types on Windows 11 I get an error message saying that it failed.

Right on.  I'll see if I can find a Windows 11 user to test de-registration.  I had taken a brief look @ that code to make sure it was undoing the changes we made during registration but my review was cursory-at-best.

**Comment by GamerzHell9137** [comment]
Checked GTK build and it fixes the associating of xci/nsp files on Windows 10. For some reason it was broken on my system.
Also successfully uninstalls it too now.

**Review by TSRBerry — COMMENTED** [review]
Could you change this to use using-statements instead of blocks?

File: src/Ryujinx.Ui.Common/Helper/FileAssociationHelper.cs [hunk 1]
```diff
@@ -1,9 +1,11 @@
+using Microsoft.IdentityModel.Tokens;
```
Comment (mmdurrant) [inline src/Ryujinx.Ui.Common/Helper/FileAssociationHelper.cs:1]: ??? Where did this come from?

**Review by TSRBerry — APPROVED** [review]
Lgtm!

**Pull request merged**